websocket/polygon_test.go

```diff
@@ -0,0 +1,29 @@
+package polygonws_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	polygonws "github.com/polygon-io/client-go/websocket"
+	"github.com/polygon-io/client-go/websocket/models"
+)
+
+func TestMain(t *testing.T) {
```

Comment:
`API_KEY=YOUR_API_KEY go test -race -v .` if you wanna try running this. this is just temporary as we figure out the best way to unit test.